internal/config/yaml_parser.go

```diff
@@ -72,7 +76,16 @@ func IsValidLogRotateConfig(config LogRotateConfig) error {
 
 func (fileCacheConfig *FileCacheConfig) validate() error {
 	if fileCacheConfig.MaxSizeMB < -1 {
-		return fmt.Errorf("the value of max-size-mb for file-cache can't be less than -1")
```

Comment:
I don't see the merit of using constants for error messages. Let's discuss this offline as well. But it's not a blocker for submission.